src/SingleBookingPage/BookingDetail.js

```diff
@@ -190,13 +190,14 @@ class BookingDetail extends React.Component<Props> {
           timeDelta && (
             <Notification hoursLeft={timeDelta} isUrgent={isUrgent} />
           )}
-        <FAQExtraInfoButton
+        {/* F.I.X.M.E Baggage is buggy */}
```

Comment:
Why `F.I.X.M.E` and not `FIXME`?

Reply:
we have `no-warning-comments` eslint rule that don't let you write `FIXME` in comments, so I use it with `F.I.X.M.E`

Reply:
:man_facepalming: